_sass/minima/_layout.scss

```diff
@@ -179,11 +179,7 @@ article.guide h1 {
 .post-content,
 article.guide {
   h1 {
-    margin-top: $spacing-unit * 1;
-
-    @include media-query(medium-up) {
```

Comment:
Any reason for removing the responsive spacing?

Reply:
The spacing isn't quite right on mobile.

<img width="387" alt="Screenshot 2021-04-13 at 23 20 02" src="https://user-images.githubusercontent.com/10757874/114653973-cb100780-9cae-11eb-9052-919b15bb5e8e.png">

I don't think it works well to have a margin-top generally when trying to space out elements vertically, as it creates unnecessary space before the first element.

I'm happy to revert this change though and pick it up in seperate PR

Reply:
Good call. But the CSS change is on H1 tags, but the title in your image is an H2 tag. Plus, you could just add the bottom margin without removing the larger top margin on desktop screen. Either way, I'll create a separate issue to address spacing, something that's come up a few times before. We should first decide that layout logic in Figma, and then port it over to CSS and test it across all pages. Created an issue [here](https://github.com/BitcoinDesign/Guide/issues/231).

Reply:
Change reverted